jsonschema2pojo-core/src/main/java/org/jsonschema2pojo/rules/ObjectRule.java

```diff
@@ -337,6 +339,7 @@ private void addHashCode(JDefinedClass jclass) {
         }
 
         for (JFieldVar fieldVar : fields.values()) {
+            if( (fieldVar.mods().getValue() & JMod.STATIC) == JMod.STATIC) continue;
```

Comment:
When squashing this PR, bring all of the changes like this into a single commit, before the actual feature.